core/templates/pages/exploration-player-page/services/audio-translation-language.service.ts

```diff
@@ -42,10 +42,10 @@ export class AudioTranslationLanguageService {
     private browserCheckerService: BrowserCheckerService,
     private languageUtilService: LanguageUtilService) {}
 
-  _currentAudioLanguageCode: string = null;
-  _allAudioLanguageCodesInExploration: string[] = null;
-  _explorationLanguageCode: string = null;
-  _automaticTextToSpeechEnabled: boolean = null;
+  _currentAudioLanguageCode: string = '';
```

Comment:
I don't think this is the correct approach, I mean for list and booleans I can see why the defaults should be `[]` and `false` but for strings, I think it is better to either:
1. Refactor the code from the callsites so that `null` is not needed.
2. Allow `null` in the type if 1. is no possible or would require too big refactor.

Reply:
yes, it wasn't the correct approach...`null` is needed for `currentAudioLanguageCode` since it is used to check if it has been set or not. The code is updated now